diff --git a/node_modules/react-native-tts/android/build.gradle b/node_modules/react-native-tts/android/build.gradle
index dbbe046..809c251 100644
--- a/node_modules/react-native-tts/android/build.gradle
+++ b/node_modules/react-native-tts/android/build.gradle
@@ -4,7 +4,9 @@ def safeExtGet(prop, fallback) {
 
 buildscript {
     repositories {
-        jcenter()
+        google()
+        mavenCentral()
+        // jcenter() removido - deprecado en Gradle moderno
     }
 
     dependencies {
